### PR TITLE
REVIEW: XMPP Newsletter November 2021

### PR DESCRIPTION
**Pull request by Echolon**
I created the draft for the next release covering **November 2021**. 

**The last release for this year! After our editorial break we will be back in February 2022!**

You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

- [You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links :pray: 
- Just paste your prepared text in this thread below or in our XMPP channel (xmpp:commteam@muc.xmpp.org?join)
- [Make a pull request by editing the file in this repository](https://github.com/xsf/xmpp.org/blob/newsletter-november2021/content/blog/2021-12-05.md)

:arrow_right: Please do not just drop bare links - rather try to make a short and decent formulated description (**<5 lines**) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on! 

**Note: If you create a pull request, which is highly appreciated, please @mention me once your PR is created!** :pray: 

XMPP Newsletter Timeline

---

This is our planed timeline for this draft:

Status:
- [x] Collection of news
- [ ] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

File: content/blog/2021-12-05.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+Sam Whited gave a [talk for the XMPP Office Hours](https://www.youtube.com/watch?v=lprIwxyPY2E) about the XSF's new fiscal hosting service!
+
+Guest talk at Berlin XMPP Meetup: [Diving deep into Briar at the XMPP Meetup Berlin](https://nico.dorfbrunnen.eu/posts/2021/diving-at-xmpp/)
+
+## Articles
+
+[PeerTube Live Chat is using XMPP](https://joinpeertube.org/news#live-plugin-app): Give your instance's audience the ability to chat during live streams!
+
+Using XMPP instead of ActivityPub? Well, [openEngiadina decided to do so](https://inqlab.net/2021-11-12-openengiadina-from-activitypub-to-xmpp.html)!
+
```
Comment (nicfab) [inline content/blog/2021-12-05.md:57]: [XMPP is a solution that allows users to have control over their personal information: our choice Snikket](https://www.nicfab.it/en/xmpp-is-a-solution-that-allows-users-to-have-control-over-their-personal-information-our-choice-snikket/)

**Review by nicfab — APPROVED** [review]

File: content/blog/2021-12-05.md (hunk 1 above)
Comment (wurstsalat3000) [reply content/blog/2021-12-05.md:57]: Thanks, should be added with https://github.com/xsf/xmpp.org/pull/1012/commits/9cc292df02b51f24ba92c4b60a7b6d8e3f3103a9

**Review by Echolon — COMMENTED** [review]
Looks good!

**Comment by Echolon** [comment]
:hourglass:  Editorial closing of the XMPP Newsletter November 2021 release

:pray:  Not only in this version, but also all the previous releases this year! MANY MANY thanks! This release to xdelatour, wurstsalat3000, seveso, palm123, Nicola Fabiano, mdosch, MattJ, Licaon_Kter, Goffi, erszcz, emus, Benoît Sibaud, Anoxinon e.V., Adrien Bourmault (neox) for their help in creation, review and translation!

:heart: :fireworks: Next release is next year on 5th of February 2021! Have a goo time until then and see you soon!

**Pull request merged**